kuryr_kubernetes/controller/handlers/lbaas.py

```diff
@@ -622,12 +634,7 @@ def _sync_lbaas_loadbalancer(self, endpoints, lbaas_state, lbaas_spec):
                     security_groups_ids=lbaas_spec.security_groups_ids,
                     service_type=lbaas_spec.type,
                     provider=self._lb_provider)
-                changed = True
-            elif lbaas_state.service_pub_ip_info:
```

Comment:
Should this remain here in case there was no lb at line 606?

Reply:
when there is no lb it means that either lbaas_state.loadbalancer is empty and consequently there is no FIP or the lb and fip are cleaned up at line 610, so it looks ok.

Reference to the patch on 4.x where it was discussed:
https://review.opendev.org/c/openstack/kuryr-kubernetes/+/750214/5/kuryr_kubernetes/controller/handlers/loadbalancer.py#708

Reply:
Okay, thanks for help, I assume I should leave it as it is in this state of the PR.